### PR TITLE
Add Docker Support

### PR DESCRIPTION
**Pull request by witoff**
**Problem**
Timesketch is currently difficult to deploy in a dockerized environment, which is desirable to simplify deployments.

**Usage**
This adds docker support for simpler local development and test. Test with:

```shell 
docker-compose build
docker-compose up
# view at http://localhost:5000
``` 

**Notes**
Builds on @jessemillar's work in https://github.com/google/timesketch/pull/249.

### CODE REVIEW

**Comment by witoff** [comment]
I signed it!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Use the official Docker Hub Ubuntu 14.04 base image
+FROM ubuntu:14.04
```
Comment (berggren) [inline Dockerfile]: We wanna use 16.04 here, seeing 14.04 is getting rather old. Totally fine to do this in a later PR though.

File: timesketch/lib/utils.py [hunk 2]
```diff
@@ -60,7 +60,8 @@ def read_and_validate_csv(path):
             if u'timestamp' not in csv_header and u'datetime' in csv_header:
                 try:
                     parsed_datetime = parser.parse(row[u'datetime'])
-                    row[u'timestamp'] = int(time.mktime(parsed_datetime.timetuple()))
+                    row[u'timestamp'] = str(int(
```
Comment (berggren) [inline timesketch/lib/utils.py]: Was this meant to go into this PR?

**Comment by berggren** [comment]
Looks good! Just a couple of small things, otherwise LGTM.

File: Dockerfile (hunk 1 above)
Comment (witoff) [inline Dockerfile]: fixed

File: timesketch/lib/utils.py (hunk 2 above)
Comment (witoff) [inline timesketch/lib/utils.py]: reverted.  this came in when rebasing and may have been something jesse added in testing.  so long as it still runs locally i'll leave this out.

**Pull request closed without merging**